[2024-09-10T08:08:33.539+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T08:08:33.608+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V4.third_task scheduled__2021-10-03T00:00:00+00:00 [queued]>
[2024-09-10T08:08:33.640+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V4.third_task scheduled__2021-10-03T00:00:00+00:00 [queued]>
[2024-09-10T08:08:33.647+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T08:08:33.839+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): third_task> on 2021-10-03 00:00:00+00:00
[2024-09-10T08:08:33.857+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V4', 'third_task', 'scheduled__2021-10-03T00:00:00+00:00', '--job-id', '6073', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpa9jspzu4']
[2024-09-10T08:08:33.868+0000] {standard_task_runner.py:105} INFO - Job 6073: Subtask third_task
[2024-09-10T08:08:33.852+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=27015) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T08:08:33.873+0000] {standard_task_runner.py:72} INFO - Started process 27045 to run task
[2024-09-10T08:08:34.074+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V4.third_task scheduled__2021-10-03T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T08:08:34.353+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_V4' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2021-10-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-03T00:00:00+00:00'
[2024-09-10T08:08:34.355+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T08:08:34.391+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T08:08:34.393+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the third task!']
[2024-09-10T08:08:34.411+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T08:08:34.421+0000] {subprocess.py:93} INFO - hey, this is the third task!
[2024-09-10T08:08:34.424+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T08:08:34.518+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T08:08:34.520+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V4, task_id=third_task, run_id=scheduled__2021-10-03T00:00:00+00:00, execution_date=20211003T000000, start_date=20240910T080833, end_date=20240910T080834
[2024-09-10T08:08:34.766+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T08:08:34.810+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T08:08:34.827+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
